_episodes/05_making_repos.md

```diff
@@ -34,11 +34,11 @@ It is perfectly possible to create a Git repository on your own computer and hav
 it exist in isolation. This allows you to keep a version history of files
 that you develop within the repository, but doesn't provide a way to easily
 share this developmental history with others, or collaborate with them on code.
-By contrast, the most common way that a person uses Git is to have a local repository
-that is linked to a remote repository. A typical example here is
+The most common way that a person uses Git is instead to have a local repository
+that is linked to a remote repository. A typical example is
 where there is a remote repository on GitHub and you have a local version of the
 repository on your own computer. You keep the remote repository up-to-date by
-'pushing' updates you make to your local repository to it (we will look more at
+'pushing' updates made on your local repository (we will look more at

```

Comment:
nit: maybe `'pushing' updates made on your local repository to the remote repository` adds some clarity?